### PR TITLE
Devao 1

### PR DESCRIPTION
**Pull request by andreieab**
Building and styling

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -117,5 +117,13 @@ module.exports = {
         id: config.googleTagManagerID,
       },
     },
+    {
+      resolve: 'gatsby-plugin-web-font-loader',
+      options: {
+        typekit: {
+          id: process.env.TYPEKIT_ID,
```
Comment (b00y0h) [inline gatsby-config.js:124]: do we need approval or anything for using typekit? or does EAB have a global license?

Comment (andreieab) [reply gatsby-config.js:124]: The Adobe cloud license EAB has, includes the usage of adobe typekit. We've been using it in enrollment for a while.

Comment (andreieab) [reply gatsby-config.js:124]: Just FYI https://helpx.adobe.com/fonts/using/font-licensing.html#web-sites

**Review by b00y0h — COMMENTED** [review]
Just have that one comment about typekit @andreieab

**Review by b00y0h — APPROVED** [review]

**Pull request merged**